### PR TITLE
Fix CI by changing vibe.d CI script name

### PR DESCRIPTION
**Pull request by kubo39**
https://github.com/vibe-d/vibe.d/pull/2611#issuecomment-931362792

### CODE REVIEW

**Review by Geod24 — APPROVED** [review]

File: buildkite/build_project.sh [hunk 1]
```diff
@@ -123,24 +123,24 @@ case "$REPO_FULL_NAME" in
...
         ;;
     vibe-d/vibe.d+tests)
         remove_spurious_vibed_tests
-        VIBED_DRIVER=vibe-core PARTS=tests ./travis-ci.sh
+        VIBED_DRIVER=vibe-core PARTS=tests ./run-ci.sh
         ;;
 
     vibe-d/vibe-core+epoll)
         remove_spurious_vibe_core_tests
-        CONFIG=epoll ./travis-ci.sh
```
Comment (Geod24) [inline buildkite/build_project.sh:138]: Not renamed yet

File: buildkite/build_project.sh [hunk 2]
```diff
@@ -123,24 +123,24 @@ case "$REPO_FULL_NAME" in
...
+        CONFIG=epoll ./run-ci.sh
         ;;
 
     vibe-d/vibe-core+select)
         remove_spurious_vibe_core_tests
-        CONFIG=select ./travis-ci.sh
```
Comment (Geod24) [inline buildkite/build_project.sh:143]: Ditto

**Comment by RazvanN7** [comment]
look like `vibe-core-epoll` and `vibe-core-select` still need updating.

**Comment by Geod24** [comment]
@kubo39 : Looks like vibe-d pulls master while vibe-core pulls the latest tag.

**Comment by PetarKirov** [comment]
Thanks @kubo39!

**Pull request merged**